manifests/machineconfigcontroller/clusterrole.yaml

```diff
@@ -14,5 +14,5 @@ rules:
   resources: ["configmaps", "secrets"]
   verbs: ["*"]
 - apiGroups: ["config.openshift.io"]
-  resources: ["images"]
+  resources: ["images", "clusterversions"]
```

Comment:
Did you run `make update` to regenerate bindata if needed

Reply:
yup